src/store/mutations.ts

```diff
@@ -373,35 +378,25 @@ export const setErrors = (state: StateIF, errors: string) => {
   state.stateModel.newRequestModel.errors = [errors]
 }
 
-export const setNrResponse = (state: StateIF, data: NameRequestI): boolean => {
+export const setNrResponse = (state: StateIF, nr: NameRequestI) => {
   try {
-    state.stateModel.newRequestModel.nr = data
-    const { applicants = [] } = state.stateModel.newRequestModel.nr
+    // set NR data
+    mutateNameRequest(state, nr)
 
+    // set Applicants
+    const applicants = state.stateModel.newRequestModel.nr?.applicants || []
     if (applicants instanceof Array) {
-      // setApplicantDetails(applicants[0]) // OLD CODE
       state.stateModel.newRequestModel.applicant = { ...applicants[0] }
     } else if (applicants) {
-      // setApplicantDetails(applicants) // OLD CODE
       state.stateModel.newRequestModel.applicant = { ...applicants }
     } else {
       // applicants is null/undefined
     }
-    // TODO: have 1 mutation for nr / applicant info and put these there
-    const nrNum = state.stateModel.newRequestModel.nr?.nrNum
-    if (nrNum?.includes('NR L')) {
-      sessionStorage.setItem('BCREG-NRL', nrNum)
-      sessionStorage.setItem('BCREG-nrNum', null)
-    } else if (nrNum) {
-      sessionStorage.setItem('BCREG-NRL', null)
-      sessionStorage.setItem('BCREG-nrNum', nrNum)
-    }
-    sessionStorage.setItem('BCREG-emailAddress', state.stateModel.newRequestModel.applicant?.emailAddress)
-    sessionStorage.setItem('BCREG-phoneNumber', state.stateModel.newRequestModel.applicant?.phoneNumber)
-    return true
```

Comment:
Return value was unused.

Reply:
The above code block was setting the values used by the interceptor to apply the request headers previously i think? 
Is this not required or done in some other way now?

Reply:
I'm not aware that it's done differently now. @kialj876 ?

Because I now call `mutateNameRequest()` and `populateApplicateData()`, which themselves set the session storage, I removed it from here.

Reply:
This looks good to me. Nice work cleaning this up!

Reply:
Sounds good, resolving!